recipe_batches/recipe_batches.py

```diff
@@ -2,13 +2,26 @@
 
 import math
 
+
 def recipe_batches(recipe, ingredients):
-  pass 
+
+    batches = 100000
```

Comment:
I think this is what you were going for
# positive infinity
p_inf = float("inf")

(2 min read)
https://medium.com/@sanatinia/how-to-work-with-infinity-in-python-337fb3987f06